vignettes/deploy-to-aks/deploy-to-aks.Rmd

```diff
@@ -0,0 +1,144 @@
+vignette: >
+  %\VignetteIndexEntry{Deploying to AKS with Azure ML SDL for R}
+  %\VignetteEngine{knitr::rmarkdown}
+  \use_package{UTF-8}
+---
+
+This article demonstrates how to deploy a model and conduct inferencing in the model using AzureML SDK for R. We will take a previous saved model. The saved model was trained with the Iris Dataset and can be used to determine if a flower is one of three Iris flower species (setosa, versicolor, virginica).
+
+We will show you how to:
+
+0. Set up the experiment
+
+1. Register a model on Azure ML
+
+2. Create the deployment environment
+
+3. Create the inferencing configurations
+
+4. Provision an AKS cluster 
+
+5. Deploy the model as a service to the AKS cluster
+
+6. Conduct inferencing on the deployed service.
+
+## 0. Set up the experiment
+
+Let's prepare for training by loading the required package, initializing a workspace, and creating an experiment.
+
+### Import package
+```{r Import package, eval=FALSE}
+library("azureml")
+```
+
+### Initialize a workspace
+
+The `Workspace` is the top-level resource for the service. It provides us with a centralized place to work with all the artifacts we will create. 
+
+You can create a `Workspace` object from a local `config.json` file
+```{r Load a workspace, eval=FALSE}
+ws <- load_workspace_from_config()
+```
+
+Or load an existing workspace from your Azure Machine Learning account
+```{r Initialize a workspace, eval=FALSE}
+ws <- get_workspace("<your workspace name>", "<your subscription ID>", "<your resource group>")
+```
+
+## 1. Register the model
+
+For this example, register an existing trained model, add name and description.
+
+```{r Register the model, eval=FALSE}
+model <- register_model(ws, 
+                        model_path = "model.rds", 
+                        model_name = "iris_model",
+                        description = "Predict an Iris flower type")
+```
+
+## 2. Create environment
+Create an environment that the model will be deployed with
+```{r Create environment, eval=FALSE}
+r_env <- r_environment(name = "r_env")
+```
+
+## 3. Create inference config
+Create the inference config that will be used when deploying the model.
+
+``` {r Create inference config, eval=FALSE}
+inference_config <- inference_config(
+  entry_script = "score.R",
+  source_directory = ".",
+  environment = r_env)
+
+```
+
+## 4. Provision AKS cluster
+This is a one time setup. You can reuse this cluster for multiple deployments after it has been created. If you delete the cluster or the resource group that contains it, then you would have to recreate it.
+
+``` {r Provision AKS cluster, eval=FALSE}
+aks_target <- create_aks_compute(ws, cluster_name = 'my-new-cluster')
+wait_for_provisioning_completion(aks_target)
+```
+
+## 5. Deploy the model to AKS and waiting for deployment
+
+Set the web service configuration (using default here)
+
+``` {r Web service configuration, eval=FALSE}
+aks_config <- aks_webservice_deployment_config()
```

Comment:
can you create aks config with these parameters: `cpu_cores = 1, memory_gb = 1` to showcase using those params. python sdk does the same